src/components/ui/FishRow.jsx

```diff
@@ -24,13 +25,13 @@ class FishRow extends React.Component {
   };
 
   render() {
-    const { fish, mkCell } = this.props;
+    const { fish, mkCell, profMulti } = this.props;
```

Comment:
I would give this a full name of `professionMultiplier` so that it's more clear in context